app-emulation/wine-staging/wine-staging-6.12-r1.ebuild

```diff
@@ -0,0 +1,666 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
```

Comment:
Do we really need a revbump for this? If the symptom is compile failure, I don't think so

Reply:
It depends on how much we care about installed `winegcc`'s behaviour. Mechanically we do change installed driver's behaviour. Perhaps nobody uses it with multiple `=` options if we don't have reports yet. I reverted the revbump.

Reply:
Thanks for explanation. I don't think many people use winegcc, so I'm fine with no revbump